src/styles/core/colors/_dark.scss

```diff
@@ -86,4 +86,6 @@
   --color-syntax-urls: rgb(102, 153, 255);
   --color-tutorial-background: var(--color-fill-tertiary);
   --color-navigator-item-hover: #{change-color(dark-color(fill-blue), $alpha: 0.5)};
+
+  --color-card-shadow: rgb(255, 255, 255, 0.04);
```

Comment:
Isn't the same as: `#{change-color(dark-color(figure-gray), $alpha: 0.4)};` ?

Reply:
oh lol could be, I just did some of those in a hurry based on design specs